internal/services/models.go

```diff
@@ -24,8 +24,8 @@ type dataStore struct {
 
 // Service is the data structure that models a service entity persisted to a database via gorm
 type Service struct {
-	ID        int
-	Name      string
+	ID        int    `gorm:"primaryKey" faker:"unique"`
+	Name      string `gorm:"not null;default:null"`
```

Comment:
Shame that gorm doesn't have a more semantic "required" thing, at least their doc page is good